iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadTsFileNode.java

```diff
@@ -103,6 +103,11 @@ public List<WritePlanNode> splitByPartition(Analysis analysis) {
               statement.isDeleteAfterLoad(),
               statement.getWritePointCount(i)));
     }
+
+    for (int i = 0; i < statement.getFailedTsFiles().size(); i++) {
+      res.add(
+          new LoadSingleTsFileNode(getPlanNodeId(), statement.getFailedTsFiles().get(i), false));
```

Comment:
Why not use `for (File failedTsFile: statement.getFailedTsFiles())` here?

Reply:
I think stream maybe better :-)

Reply:
thanks homie, I'll take your advice :)